public/js/main.js

```diff
@@ -63,6 +63,24 @@ $(function() {
 	});
 
 	$('.bio-modal').on('shown.bs.modal', function () {
-  		$(this).find('textarea').focus();
-  	});
+		$(this).find('textarea').focus();
+	});
+
+	$('[data-clipboard]').each(function (i, element) {
+		$(element).tooltip({
+			trigger: 'manual',
+			placement: 'bottom',
+			title: 'Copied!'
+		});
+
+		var client = new ZeroClipboard(element);
+		client.on( "ready", function( readyEvent ) {
```

Comment:
Could you drop all the extra spaces here?

I need to document my JS styles. It's based on something called idiomatic JS, but it's basically the PSRs applied to JS.

```javascript
		client.on("ready", function (readyEvent ) {
			client.on("aftercopy", function (event) {
// etc.
```